### ISSUE TITLE
tutorial not working: Text classification with Scikit-Learn

### ISSUE THREAD

**Issue by az0** [heartexlabs/label-studio#6338]
**Describe the bug**

I am following the official tutorial for a simple [text classifier](https://labelstud.io/tutorials/sklearn-text-classifier) , but `label-studio start` causes crash. Also, the label-studio-ml development log shows errors when labeling, training, or requesting predictions.

**To Reproduce**

1. I followed the steps in the official [text classifier](https://labelstud.io/tutorials/sklearn-text-classifier)  tutorial
3. I used the "development mode" backend option
4. When running `label-studio start my_project --init --ml-backends http://localhost:9090`, I got a traceback (see attachment), so instead, I created a project myself in the web interface
6. I imported a CSV file with 32 short texts. (I would appreciate the tutorial providing a sample file.)
7. I enabled pretraining
8. I clicked "send test request" and it seemed to work. The ml backend shows a /predict request with status 200.
9. I labeled texts. When opening a record, there is a traceback. See attachment `label-studio-ml-open-for-label.log`
10. When submitting a label, there was a traceback. See attachment `label-studio-submit-label.log`
11. I clicked "start training" and got a similar traceback.
12. I checked three tasks and clicked "request predictions" and got a similar traceback

**Expected behavior**

* No traceback
* Show predictions in label-studio

**Screenshots**

[start_project.log](https://github.com/user-attachments/files/16914456/start_project.log)
[label-studio-ml-open-for-label.log](https://github.com/user-attachments/files/16914447/label-studio-ml-open-for-label.log)
[label-studio-submit-label.log](https://github.com/user-attachments/files/16914448/label-studio-submit-label.log)

**Environment (please complete the following information):**
label-studio                   1.13.1
label-studio-ml                1.0.9
label-studio-sdk               1.0.5
label-studio-tools             0.0.4
Python 3.12.3
Ubuntu 24.04 (Noble)
